scripts/ci.py

```diff
@@ -97,6 +97,18 @@ def _set_branch_cmd(opts):
     update_repo(opts.repository, opts.branch)
 
 
+def _checkout_pr_cmd(opts):
+    vprint(f"checking if PR #{opts.pr} exists for {opts.repository}")
+    try:
+        shell(f"git ls-remote origin 'pull/*/head' | grep 'refs/pull/{opts.pr}/head'")
+    except subprocess.CalledProcessError as error:
+        vprint(f"PR #{opts.pr} does not exist. Please specify a valid PR number. {error}")
+        exit(1)
```

Comment:
**Nit**: Seems like this error message should be displayed regardless of verbosity level (i.e., `print` instead of `vprint`).